packages/core/src/StrykerCli.ts

```diff
@@ -79,6 +79,11 @@ export default class StrykerCli {
       .option('--testRunner <name>', 'The name of the test runner you want to use')
       .option('--reporters <name>', 'A comma separated list of the names of the reporter(s) you want to use', list)
       .option('--plugins <listOfPlugins>', 'A list of plugins you want stryker to load (`require`).', list)
+      .option(
+        '--appendPlugins <listOfPlugions>',
+        'A list of additional plugins you want Stryker to load (`require`) without overwriting the (default) `plugins`.',
```

Comment:
Why is this description so different from the one on `stryker-core.json`? I usually try to keep both explanations and the `readme` in sync.